.github/actions/c-filedemo/action.yml

```diff
@@ -0,0 +1,27 @@
+name: Run filedemo
```

Comment:
We are moving away from this "actions" approach to an approach using a python script wrapper in the other repos. Granted, those have more complicated & multiple test cases, but it might makes sense to do the same here? I see both sides so I'm happy either way.

Porcupine eg: https://github.com/Picovoice/porcupine/pull/890/files

Reply:
I see, if we are going to do the same thing for other repos, then it makes sense to also do it for Koala.